terraform/gcp/featureform/featureform.tf

```diff
@@ -0,0 +1,42 @@
+provider "helm" {
+  kubernetes {
+    config_path = "~/.kube/config"
+  }
+}
+
+resource "helm_release" "certmgr" {
+  name = "cert-mgr"
+  namespace = "default"
+  version = var.cert_manager_version
+
+  repository = "https://charts.jetstack.io"
+  chart = "cert-manager"
+  
+  set {
+    name  = "installCRDs"
+    value = "true"
+  }
+}
+
+resource "helm_release" "featureform" {
+  name = "featureform"
+  namespace = "default"
+  version = "v0.1.0"
```

Comment:
Do we need to set this? Can it just pull the latest version?

Reply:
It isn't needed; I will remove it.